inspirehep/modules/search/walkers/elasticsearch.py

```diff
@@ -27,6 +27,8 @@
 
```

Comment:
It would be cool to add a preparatory commit that adds the `__future__` block and fixes the license. See: https://github.com/inspirehep/inspire-next/pull/1500/commits/e399aa155af1131c222c54909e671df630fe3d4d